android-vjestina-homework-movie-app-data-mock/app/src/main/java/agency/five/codebase/android/movieapp/data/database/DbFavoriteMovie.kt

```diff
@@ -0,0 +1,13 @@
+package agency.five.codebase.android.movieapp.data.database
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity
+data class DbFavoriteMovie(
```

Comment:
When I set tableName property my project crashes upon launch, example of setting and using it is attached
![image](https://user-images.githubusercontent.com/73428201/209673613-c9dc1a4d-3f34-4a28-9c2a-62ed8a34520e.png)
![image](https://user-images.githubusercontent.com/73428201/209673644-07c15636-8a84-4c97-b258-7dec630a7b1c.png)

Reply:
That's because you are changing schema of a database by changing the column name. 
You should increase database version and provide a migration. 
To avoid this you can just increase version and add `.fallbackToDestructiveMigration()` to `RoomDatabase.Builder` before building the database.